pkg/kubeapiserver/options/authentication.go

```diff
@@ -51,7 +51,8 @@ type AnonymousAuthenticationOptions struct {
 }
 
 type BootstrapTokenAuthenticationOptions struct {
-	Allow bool
+	Enable                       bool
+	DeprecatedEnableExperimental bool
```

Comment:
Thanks, I was looking for something like `fs.MarkDeprecated` but didn't find it. This is much better.